controller/array_action/array_mediator_svc.py

```diff
@@ -283,21 +293,25 @@ def _get_wwn_by_volume_name_if_exists(self, volume_name):
         logger.debug("found wwn : {0}".format(wwn))
         return wwn
 
-    def _get_volume_name_by_wwn_if_exists(self, volume_id):
+    def _get_cli_volume_by_wwn_if_exist(self, volume_id):
```

Comment:
to distinguish an svc volume object from our cross-storage volume object

Reply:
I will close this conversation though I would like us to open a ticket to rename the phrase "cli".
Arbel - please open a ticket.

Reply:
could you clarify why/a better alternative?

Reply:
CLI refer to command line interface. Although it is the medium that bring back the objects, it is not the right description for the object itself. It can be storae_object_volume or something else.
We don't need to decide it now - just open a ticket, talk it through a session and change accordingly.
Arbel - please open a ticket.

Reply:
CSI-2342